src/bin/inx-chronicle/stardust_inx/error.rs

```diff
@@ -16,12 +16,6 @@ pub enum InxWorkerError {
     InfluxDb(#[from] influxdb::Error),
     #[error("expected INX address with format `http://<address>:<port>`, but found `{0}`")]
     InvalidAddress(String),
-    #[error("wrong number of ledger updates: `{received}` but expected `{expected}`")]
```

Comment:
I think we should leave these until we move the error and make sure we don't need them.